### PR TITLE
Users/desomaga/wiki enhance

### PR DESCRIPTION
**Pull request by deepika791**
App Configuration changes

### CODE REVIEW

File: NotificationService/NotificationHandler/appsettings.json [hunk 1]
```diff
@@ -1,71 +1,9 @@
...
-    "Uri": "__CosmosDBUri__",
-    "Key": "__CosmosDBKey__",
-    "Database": "__CosmosDBDatabase__",
-    "Container": "__CosmosDBContainer__"
-  },
-  "StorageAccount": {
-    "ConnectionString": "__StorageAccountConnectionString__",
-    "BlobContainerName": "__BlobContainerName__",
-    "MailTemplateTableName": "__MailTemplateTableName__"
+    "SecretUri": "__KeyVaultSecretUri__"
```
Comment (reddaiahnethi) [inline NotificationService/NotificationHandler/appsettings.json:4]: can you change this to keyvault url

File: NotificationService/NotificationHandler/appsettings.json [hunk 2]
```diff
@@ -1,71 +1,9 @@
...
   },
-  "StorageType": "__StorageType__",
-  "NotificationProviderType": "__NotificationProviderType__"
+  "AppConfig": {
+    "ForceRefresh": ".appconfig.featureflag/ForceRefresh"
```
Comment (reddaiahnethi) [inline NotificationService/NotificationHandler/appsettings.json:7]: why do we need featureflag?

File: NotificationService/NotificationHandler/appsettings.json (hunk 1 above)
Comment (deepika791) [reply NotificationService/NotificationHandler/appsettings.json]: changed to keyvaultUrl

File: NotificationService/NotificationHandler/appsettings.json (hunk 2 above)
Comment (deepika791) [reply NotificationService/NotificationHandler/appsettings.json]: removing that as there is a automatic refresh of 30sec

**Review by harshitagarwal93 — APPROVED** [review]

**Review by sanjayprajap — DISMISSED** [review]

**Review by sanjayprajap — DISMISSED** [review]

**Review by sanjayprajap — APPROVED** [review]

**Review by sanjayprajap — APPROVED** [review]

**Review by reddaiahnethi — APPROVED** [review]

**Pull request merged**